tests/phpunit/class-wc-stripe-webhook-handler-test.php

```
@@ -189,6 +189,60 @@ function ( $passed_order ) use ( $order ) {
+						'charges'            => (object) [
+							'total_count' => 1,
+							'data'        => [
+								(object) self::MOCK_PAYMENT_INTENT['charges']['data'][0],
+							],
+						],
+						'last_payment_error' => null,
+					],
+				],
+			];
+
+			$notification_as_arrays = json_decode( wp_json_encode( $notification ), true );
```

PHPMD (2.15.0)

[error] The variable $notification_as_arrays is not named in camelCase. (undefined)

(CamelCaseVariableName)